File: cue2tracklist/cue2tracklist.py
import re
import sys

def cue2tracklist(filepath):
    with open(filepath, 'r') as fID:
        exp = re.compile(r'\s{0,}TITLE \"(.*?)\"')
        tracklist = []
        for line in fID:
            test = exp.match(line)
            if test:
                tracklist.append(test.group(1))

    if len(tracklist) != 0:
        print('Track Listing:')
        print('')
        for track in tracklist:
            print('[#] {0}'.format(track))

if __name__ == '__main__':
    cue2tracklist(sys.argv[1])

File: cue2tracklist/cue2tracklist.m
function cue2tracklist(filepath)
fID = fopen(filepath, 'r');

regexpattern = '[ ]{1,}TITLE \"(.*?)\"';
tracklist = {};
while ~feof(fID)
    tline = fgetl(fID);
    test = regexp(tline, regexpattern, 'Tokens');
    if ~isempty(test)
        tracklist(end+1) = test{1}(1);
    end
end

fclose(fID);

fprintf('Track Listing:\n\n')
for ii = 1:length(tracklist)
    fprintf('[#] %s\n', tracklist{ii})
end

end